certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -2893,261 +2960,435 @@
           {
             "name": "body",
             "in": "body",
-            "required": false,
+            "required": true,
             "schema": {
               "type": "object",
               "properties": {
-                "name": {
+                "typeName": {
                   "type": "string",
-                  "description": "Custom Field Name",
-                  "title": "Custom Field Name"
+                  "description": "Type Name.",
+                  "title": "",
```

Comment:
@RjImpx There are many title values are empty, can you please provide valid value for to them. if you have specific reason to keep empty strings, Please let us know

Reply:
@sriyen-msft These were not visible to the user however we went ahead and updated the titles. thanks